Pascal_Triangle.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require 'vendor/autoload.php';
+
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+// Initialize template engine https://twig.symfony.com/
+$loader = new FilesystemLoader('templates');
```

Comment:
You could replase it with https://github.com/HenintsoaHARINORO/ExoPHP/blob/f2041c6a0c399a5766c5b9c1073bbcd20e2f809d/messages.php#L8
To reduce code duplication

Reply:
Why,I thought it was the same with the first  branch issues1
So I just copy paste

Reply:
Because if we want to change directory of template you will have to change it everywhere

Reply:
If the creation of this class will be moved a common file it would be simple to change the template location.